Python/Iniciante/1185.py

```diff
@@ -0,0 +1,30 @@
+# Problema 1185 - Beecrowd - Iniciante - Nível 2
+
+# Iniciamos a matriz, que será uma lista de listas, a soma 
+# se iniciará com 0 e recebmos a operação a ser realizade
+matriz = []
+soma = 0
+operacao = input()
+
+# Preenchemos a cada linha e depois, utilizamos 'append'
+# para inserir a linha na matriz
+for _ in range(12):
+    linha = []
+    for _ in range(12):
+        linha.append(float(input()))
+    matriz.append(linha)
```

Comment:
É possível evitar o uso de dois loops "for" para a criação e preenchimento da matriz utilizando [List Comprehension](https://www.geeksforgeeks.org/python-list-comprehension/), deixando o código mais enxuto

```suggestion

operacao = input()

# Criando e preenchendo a matriz utilizando List Comprehension (Compreensão de Listas)
matriz = [[float(input()) for _ in range(12)] for _ in range(12)]
soma = 0
```